tendermint/core_service.go

```diff
@@ -238,7 +239,12 @@ func verifyAndSaveConfig(defaultConfig *cfg.Config) error {
 
 func createTendermintNode(defaultConfig *cfg.Config) (*nm.Node, error) {
 	logr := log.New()
-	logr.SetOutput(io.Discard)
+	logr.SetOutput(&lumberjack.Logger{
+		Filename:   "/var/log/arcana/tm.log",
```

Comment:
Is it fair to hard-code this?

Reply:
should be fine for now, will change in upcoming version.